.bundlewatch.config.json

```diff
@@ -2,7 +2,7 @@
     "files": [
         {
             "path": "./dist/assets/common.mjs",
-            "maxSize": "1.7MB"
+            "maxSize": "1.71MB"
```

Comment:
This is just due to the small amount of new site code pushing us over 1.7, and not because of Typebox, right?

Reply:
Here is the bundle size as reported by vite with and without typebox:

Without typebox
dist/assets/common.mjs  1,743.61 kB │ gzip: 566.87 kB │ map:  8,139.42 kB

With typebox
dist/assets/common.mjs  1,780.89 kB │ gzip: 573.78 kB │ map:  8,309.57 kB

_The "without" scenario has the types neutralized as "any" and the typebox import removed. Nothing else changed._

So typebox contributes to the bundle size to the extent of ~ 7kB compressed.